### PR TITLE
updating to mdc 35

### PR DESCRIPTION
**Pull request by prateekbh**

### CODE REVIEW

**Comment by artipost** [comment]
Failed pictures (24): [failed-pictures.zip](https://gist.githubusercontent.com/artipost/7d79a0521d13b771be3793de20de8a82/raw/cf3689bececc3b61fd3d9af1c5955049a5f0d6af/failed-pictures.zip)

##
:postbox: Posted by [artipost.io](https://artipost.io), through Travis build [373713834](https://travis-ci.org/prateekbh/preact-material-components/builds/373713834), for commit [a93405b](https://github.com/prateekbh/preact-material-components/pull/787/commits/a93405b14f0e71991b61fc37373c60809bbbf664).

**Comment by prateekbh** [comment]
@filoozom @YePpHa for your review

File: Chips/index.d.ts [hunk 1]
```diff
@@ -6,6 +6,7 @@ declare interface IChipProps extends JSX.HTMLAttributes {
   children?: ChipText[];
   choice?: boolean;
   filter?: boolean;
+  entry?: boolean;
```
Comment (YePpHa) [inline Chips/index.d.ts]: This should be `input`

File: Chips/Chips.jsx [hunk 2]
```diff
@@ -7,7 +7,7 @@ class ChipSet extends MaterialComponent {
   constructor() {
     super();
     this.componentName = 'chip-set';
-    this._mdcProps = ['choice', 'filter'];
+    this._mdcProps = ['choice', 'filter', 'input'];
```
Comment (YePpHa) [inline Chips/Chips.jsx]: What about the [pre-selected](https://github.com/material-components/material-components-web/tree/master/packages/mdc-chips#pre-selected) chip?

File: TopAppBar/TopAppBar.jsx [hunk 3]
```diff
@@ -5,7 +5,7 @@ class TopAppBar extends MaterialComponent {
   constructor() {
     super();
     this.componentName = 'top-app-bar';
-    this._mdcProps = ['short', 'short-collapsed'];
+    this._mdcProps = ['short', 'short-collapsed', 'fixed'];
```
Comment (YePpHa) [inline TopAppBar/TopAppBar.jsx]: What about [prominent](https://github.com/material-components/material-components-web/tree/master/packages/mdc-top-app-bar#prominent)?

Comment (prateekbh) [reply TopAppBar/TopAppBar.jsx]: Done

File: Chips/Chips.jsx (hunk 2 above)
Comment (prateekbh) [reply Chips/Chips.jsx]: done

File: Chips/index.d.ts (hunk 1 above)
Comment (prateekbh) [reply Chips/index.d.ts]: done

**Comment by artipost** [comment]
Failed pictures (26): [failed-pictures.zip](https://gist.githubusercontent.com/artipost/fba4ac2f4b100a4379038db6700e09a4/raw/516f4e6aa4dad727a4340ef67a5cb74bda0a9b00/failed-pictures.zip)

##
:postbox: Posted by [artipost.io](https://artipost.io), through Travis build [374205039](https://travis-ci.org/prateekbh/preact-material-components/builds/374205039), for commit [4db9a3a](https://github.com/prateekbh/preact-material-components/pull/787/commits/4db9a3a919d9551b1dd15e60f9d2958f2912088b).

**Comment by artipost** [comment]
Failed pictures (1): [failed-pictures.zip](https://gist.githubusercontent.com/artipost/b2fbc73adfd9e31f35d108caac52732d/raw/106cafa9726e073e15a6010b7b0d8ffc50c1b1a7/failed-pictures.zip)

##
:postbox: Posted by [artipost.io](https://artipost.io), through Travis build [374209371](https://travis-ci.org/prateekbh/preact-material-components/builds/374209371), for commit [232bcf2](https://github.com/prateekbh/preact-material-components/pull/787/commits/232bcf26c05b183cb2449ee17346260de56b037b).

**Comment by artipost** [comment]
Failed pictures (2): [failed-pictures.zip](https://gist.githubusercontent.com/artipost/fe259799cc72d638fad32d976d609e8b/raw/13559207f6a27e7c4c6074c2cc4dbe4ec83f35e9/failed-pictures.zip)

##
:postbox: Posted by [artipost.io](https://artipost.io), through Travis build [374220841](https://travis-ci.org/prateekbh/preact-material-components/builds/374220841), for commit [5359f68](https://github.com/prateekbh/preact-material-components/pull/787/commits/5359f687f2174bc52b400bf61dc1c9bfe0e86aff).

**Comment by artipost** [comment]
Failed pictures (2): [failed-pictures.zip](https://gist.githubusercontent.com/artipost/76130ddfdb1b619119d3821608f4515e/raw/1574c0a500bb443971c2861272a4e5f7151f8e53/failed-pictures.zip)

##
:postbox: Posted by [artipost.io](https://artipost.io), through Travis build [374468238](https://travis-ci.org/prateekbh/preact-material-components/builds/374468238), for commit [0d6104f](https://github.com/prateekbh/preact-material-components/pull/787/commits/0d6104fde2a23a3a3f073c0ca479dc9662bc2fdc).

**Comment by artipost** [comment]
Failed pictures (2): [failed-pictures.zip](https://gist.githubusercontent.com/artipost/514274d0d44a0be43cdd08aff8957750/raw/37cb9ef06046d83cf718e9c075d2a75c05f09e52/failed-pictures.zip)

##
:postbox: Posted by [artipost.io](https://artipost.io), through Travis build [374470964](https://travis-ci.org/prateekbh/preact-material-components/builds/374470964), for commit [c488497](https://github.com/prateekbh/preact-material-components/pull/787/commits/c4884970d5ba1314a0baa65b4a8d0d7584451289).

**Pull request merged**